src/main/java/org/spburegistry/backend/dto/ProjectTO.java

```diff
@@ -0,0 +1,51 @@
+package org.spburegistry.backend.dto;
+
+import java.util.Date;
+import java.util.Set;
+
+import org.spburegistry.backend.enums.WorkFormat;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ProjectTO {
+    private Long projectId;
+
+    private String name;
+
+    private String description;
+
+    private String requirements;
+
+    private ClientTO client;
+
+    private Set<StudentTO> students;
+
+    private Set<ClinicTO> clinics;
+
+    private Set<String> tags;
```

Comment:
I think it is better to add TagTO and find tags by id

Reply:
Done